openedx/features/caliper_tracking/tests/expected/xmodule.partitions.assigned_user_to_partition.json

```
@@ -0,0 +1,53 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Linked",
+  "actor": {
+    "id": "http://localhost:18000",
+    "type": "SoftwareApplication"
```

```suggestion
    "name": "edX"
```
wouldn't it be better ?